spec/factories/mailing_list/contact_factory.rb

```diff
@@ -1,6 +1,5 @@
 FactoryBot.define do
-  factory :mailing_list_contact, class: MailingList::Steps::Contact do
```

Comment:
This should have the factory file renamed as well

Reply:
Ah good spot